src/components/Hero.tsx

```diff
@@ -0,0 +1,118 @@
+  const [results, setResults] = useState<PostResult[]>([]);
+  const [hasFocus, setFocus] = useState<boolean>(false);
+  
+  
+  useEffect(() => {
+    if (debouncedSearchTerm) {
+
+      fetchWithSearch("stars", 3, debouncedSearchTerm).then((results) => {
+        setResults(results as unknown as []);
+      });
+    } else {
+      setResults([]);
+    }
+  }, [debouncedSearchTerm])
+  
+
+  return (
+    <div className='flex flex-col py-[95px] items-center mx-[10px] '>
+        <div>
+          <h1 className='font-Lexend text-4xl md:text-5xl text-center text-lightSlate leading-tight tracking-tight'>
+            Find <span className='bg-gradient-to-r from-gradFirst via-gradMiddle to-gradLast bg-clip-text text-transparent' >Open-Source Repositories</span> <br/> to contribute today
+          </h1>
+        </div>
+        <div className='mt-[45px] px-[15px] gap-x-[10px] py-[10px] justify-between bg-white shadow-[0_35px_60px_-15px_rgba(0,0,0,0.3)] rounded-[16px] md:min-w-[422px]  flex'>
+          <img src={searchNormal} alt="search icon" />
+          <input
+            onFocus={() => setFocus(true)}
+            onBlur={() =>
+              setTimeout(() => {
+                setFocus(false);
+              }, 200)
+            }
+            onChange={(e => setSearchTerm(e.target.value))}
+            type="text" placeholder='Search repositories'
+            className='w-full outline-none text-[16px] text-lightSlate'
+          />
+          {/* todo: implement keyboard shortcut to bring the input field in focus - issue #205 */}
+          {/* <img className='pt-[7px]' src={cmdK} alt="command k" /> */}
+        </div>
+        <div className='mt-[10px] flex w-full justify-center relative'>
+          { results.length > 0 && hasFocus && 
+            <div className='flex md:min-w-[400px] pb-[8px] absolute z-50 max-w-[400px] flex-col bg-white rounded-[10px]'>
+              <div className='bg-gray-100 py-[10px] px-[10px] md:px-[15px] border-b-gray-100 border-b-[2px] rounded-[10px] rounded-b-none w-full'>
+                <p className='text-gray-500 text-[14px] font-semibold'>Repository</p>
+              </div>
+
+              {
+                results.map( ({full_name, description, issues, stars, user_id }) => (
+                  <a
+                  key={user_id}
+                  href={`https://app.opensauced.pizza/repos/${full_name}`}
+                  rel="noreferrer"
+                  target="_blank"
+                  >
+                    <div  className='flex flex-col hover:bg-gray-50 '>
+                      <div className='flex flex-col px-[10px] md:px-[15px] py-[10px]'>
+                        <div className='flex items-center gap-x-[10px] mb-[5px]'>
+                          <div className='w-[25px] h-[25px] overflow-hidden border-gray-400 border-[1px] bg-red-100  rounded-full'>
+                            <img className='w-full h-full' src={`https://avatars.githubusercontent.com/u/${user_id}`} alt="user avatar" />
```

Comment:
please change the alt text here as well

Reply:
@0-vortex, what do you think, what should be the alt text here?

Reply:
> Either of them is fine:
> 
> 
> ```
> 
> alt={`GitHub user avatar for ${full_name}`} />
> 
> alt={full_name}
> 
> ```